### PR TITLE
add yelp_review_full dataset

### PR DESCRIPTION
**Pull request by hfawaz**
This corresponds to the Yelp-5 requested in https://github.com/huggingface/datasets/issues/353

### CODE REVIEW

File: datasets/yelp_review_full/yelp_review_full.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        my_urls = _URLs[self.config.name]
+        data_dir = dl_manager.download_and_extract(my_urls)
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": os.path.join(data_dir, "yelp_review_full_csv/train.csv"),
```
Comment (lhoestq) [inline datasets/yelp_review_full/yelp_review_full.py:106]:
```suggestion
                    "filepath": os.path.join(data_dir, "yelp_review_full_csv", "train.csv"),
```

File: datasets/yelp_review_full/yelp_review_full.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                    "split": "train",
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={"filepath": os.path.join(data_dir, "yelp_review_full_csv/test.csv"), "split": "test"},
```
Comment (lhoestq) [inline datasets/yelp_review_full/yelp_review_full.py:112]:
```suggestion
                gen_kwargs={"filepath": os.path.join(data_dir, "yelp_review_full_csv", "test.csv"), "split": "test"},
```

**Review by lhoestq — COMMENTED** [review]
Really nice thank you !

Could you also add the tags of the dataset card ? It's the yaml part at the top of the dataset card. You can find more info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

**Comment by hfawaz** [comment]
Moved to https://github.com/huggingface/datasets/pull/1315

**Pull request closed without merging**